### ISSUE TITLE
v1.3 fails to install the .so

### ISSUE THREAD

**Issue by kattjevfel** [jbruchon/libjodycode#3]
As of v1.3, libjodycode tries to install `libjodycode.so.`, missing the last digit:
```
katt@main build/libjodycode (master %) » l test/usr/local/lib
total 108K
-rw-r--r-- 1 katt katt 108K May  7 12:13 libjodycode.a
lrwxrwxrwx 1 katt katt   15 May  7 12:13 libjodycode.so -> libjodycode.so.
lrwxrwxrwx 1 katt katt   15 May  7 12:13 libjodycode.so. -> libjodycode.so.
```

This might be related to all the grep warnings in the makefile, which includes the one one for $VERSION, but the same happens for v1.2 which does install properly.

v1.3:
```
katt@main build/libjodycode (master) » make DESTDIR=test install
test -e test/usr/local/lib || mkdir -p test/usr/local/lib
test -e test/usr/local/include || mkdir -p test/usr/local/include
test -e test/usr/local/share/man/man7 || mkdir -p test/usr/local/share/man/man7
grep: warning: stray \ before #
grep: warning: stray \ before #
grep: warning: stray \ before #
grep: warning: stray \ before #
install -m 0644 libjodycode.so            test/usr/local/lib/libjodycode.so.
ln -sf          libjodycode.so. test/usr/local/lib/libjodycode.so.
ln -sf          libjodycode.so. test/usr/local/lib/libjodycode.so
install -m 0644 libjodycode.a             test/usr/local/lib/libjodycode.a
install -m 0644 libjodycode.h             test/usr/local/include/libjodycode.h
install -m 0644 libjodycode.7             test/usr/local/share/man/man7/libjodycode.7
```

v1.2:
```
katt@main build/libjodycode » make DESTDIR=test install
cc -shared -o libjodycode.so cacheinfo.o jody_hash.o oom.o paths.o size_suffix.o sort.o string.o string_malloc.o strtoepoch.o version.o win_stat.o win_unicode.o  -Wl,-soname,libjodycode.so.1
ar rcs libjodycode.a cacheinfo.o jody_hash.o oom.o paths.o size_suffix.o sort.o string.o string_malloc.o strtoepoch.o version.o win_stat.o win_unicode.o 
test -e test/usr/local/lib || mkdir -p test/usr/local/lib
test -e test/usr/local/include || mkdir -p test/usr/local/include
test -e test/usr/local/share/man/man7 || mkdir -p test/usr/local/share/man/man7
grep: warning: stray \ before #
grep: warning: stray \ before #
grep: warning: stray \ before #
grep: warning: stray \ before #
install -m 0644 libjodycode.so            test/usr/local/lib/libjodycode.so.1.2
ln -sf          libjodycode.so.1.2 test/usr/local/lib/libjodycode.so.1
ln -sf          libjodycode.so.1.2 test/usr/local/lib/libjodycode.so
install -m 0644 libjodycode.a             test/usr/local/lib/libjodycode.a
install -m 0644 libjodycode.h             test/usr/local/include/libjodycode.h
install -m 0644 libjodycode.7             test/usr/local/share/man/man7/libjodycode.7
```

I might also be on the completely wrong track, but this is what I could come up with.

Arch Linux
gcc 13.1.1
make 4.4.1

**Comment by jbruchon** [comment]
I am not getting the same thing with GNU Make 4.3, but I have just pushed something that removes the offending characters and works in my testing. Check latest master?

**Comment by kattjevfel** [comment]
Updated to bd724ba6862b1698472e224e64bb8009190985b3 but still getting the same thing. I decided to stop being lazy and ran git bisect and kind of unsurprisingly it points to 4b45d37c6179f7a96e65a23e4a64d70daedc2dd5 being the first bad commit.

Looking at that commit, you've changed it to look for `LIBJODYCODE_VER` in your chroot_build.sh, but not the makefile, where it still looks for `#define VER`.
